steps/AddLang.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+using NUnit.Framework;
+
+namespace MarsQA_1.Feature
+{
+    [Binding]
+    class AddLang
+    {
+
+        [Given(@"I add language and level")]
+        public void GivenIAddLanguageAndLevel()
+        {
+            Pages.language.Add_Language();

```

Comment:
You could instantiate the Page class and call it's methods multiple times instead of adding the class on every single step